Einbinden der JavaScript- und CSS-Dateien (#24)
fossgis/karte.openstreetmap.de: README.md

biome check --fix
```

## Libraries

> Including many JavaScript and CSS files with `<script>` and `<link>` tags can make `index.html` messy. Instead, we use `@import` in `index.css` and `import` in `index.js`, so that only these two files need to be included in the HTML.
> 
> ### External libraries
> 
> External libraries shall be downloaded to `public/lib/<add-official-name-of-library>` and added to the git repo.

Reviewer: ```suggestion
External libraries shall be downloaded to `public/lib/<add-official-name-of-library>` and added to the git repo. JavaScript/CSS files are included directly in `index.js` and `index.css`.
```
